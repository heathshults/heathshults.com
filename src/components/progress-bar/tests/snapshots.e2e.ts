import { snapIt } from '../../../test';

const component = 'progress';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('basic progress bar', '<hs-progress percentage="5">5%</hs-progress>');
    snap('size correctly', '<hs-progress size="small" percentage="15">15%</hs-progress>');
    snap('type correctly', '<hs-progress size="medium" type="info" percentage="20">20%</hs-progress>');
    snap(
      'rounded ends correctly',
      '<hs-progress size="large" type="success" percentage="25" rounded>25%</hs-progress>'
    );
  });
});
