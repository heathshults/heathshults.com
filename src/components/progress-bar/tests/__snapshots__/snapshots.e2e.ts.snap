// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`progress renders basic progress bar 1`] = `"<hs-progress percentage=\\"5\\" class=\\"hydrated\\"><div class=\\"hs-progress\\">5%</div></hs-progress>"`;

exports[`progress renders rounded ends correctly 1`] = `"<hs-progress size=\\"large\\" type=\\"success\\" percentage=\\"25\\" rounded=\\"\\" class=\\"hydrated\\"><div class=\\"hs-progress hs-progress--rounded u-large\\">25%</div></hs-progress>"`;

exports[`progress renders size correctly 1`] = `"<hs-progress size=\\"small\\" percentage=\\"15\\" class=\\"hydrated\\"><div class=\\"hs-progress u-small\\">15%</div></hs-progress>"`;

exports[`progress renders type correctly 1`] = `"<hs-progress size=\\"medium\\" type=\\"info\\" percentage=\\"20\\" class=\\"hydrated\\"><div class=\\"hs-progress u-medium\\">20%</div></hs-progress>"`;
